src/externs/js-cookie.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @fileoverview js-cookie library extern.
+ */
+
+/**
+ * @constructor
+ * @return {!Cookies}
+ */
+function Cookies() {}
```

Comment:
This seems to be a namespace as well? If you never call "new Cookies()" then it's probably a namespace.